src/components/login/LoginForm.js

```diff
@@ -29,13 +30,20 @@ const LoginForm = (props) => {
                     value ={password}
                     onChange={passwordChange}
                 />
-                <label className="active" for="password">Password</label>
```

Comment:
Return this label, I see you have commented it out

Reply:
Worked on it